src/controller/AutoCommissioner.cpp

```diff
@@ -221,6 +221,23 @@ CommissioningStage AutoCommissioner::GetNextCommissioningStage(CommissioningStag
     return CommissioningStage::kError;
 }
 
+EndpointId AutoCommissioner::GetEndpoint(const CommissioningStage & stage)
+{
+    switch (stage)
+    {
+    case CommissioningStage::kWiFiNetworkSetup:
```

Comment:
Just wondering how it is working for simple ETH and future 4G/5G network support, since it is not referenced here.. ?

Reply:
kEthernetNetworkEnable is not usable ? Imagine a device that want to save power ..

Reply:
Issue https://github.com/project-chip/connectedhomeip/issues/14725 to track